CenterofExcellenceALMAccelerator/SolutionPackage/CanvasApps/cat_poweropsdevopsedition_7f3f4_DocumentUri_msapp_src/Src/DeploymentSettingsScreen.fx.yaml

```diff
@@ -3513,6 +3532,13 @@ DeploymentSettingsScreen As screen:
                         UpdateContext({EnvironmentDeploymentSettings: ParseDeploymentSettings.Run("[]")}),
                         UpdateContext({EnvironmentDeploymentSettings: ParseDeploymentSettings.Run(locEnvironmentDeploymentSetting.Data)})
                     );
+                	// Read SolutionUpgrade tag
```

Comment:
If refactored above this would be 

                	UpdateContext({solutionupgradetag: 
                			 LookUp(
                                        EnvironmentDeploymentSettings,
                                        Name = "TriggerSolutionUpgrade"
                                    ).Value
                			});

Reply:
Refractored the code by setting true/false instead of upgrade/update.